trio/__init__.py

```diff
@@ -1,5 +1,6 @@
 """Trio - A friendly Python library for async concurrency and I/O
 """
+from __future__ import annotations  # isort: skip
```

Comment:
I don't think you should have to tell isort to skip a `__future__` import

Reply:
You shouldn't no, it's because of the `skip` directive on the next import. But using `split` instead behaves more correctly.